### PR TITLE
Add the function to delete and edit recipes.

### PR DESCRIPTION
**Pull request by TuanDinhLe**

### CODE REVIEW

File: server/utils/recipeUtils.js [hunk 1]
```diff
@@ -15,13 +15,57 @@ const getByName = asyncHandler(async (recipeName, userID, Recipe) => {
     return recipe
 })
 
-const getByID = asyncHandler(async (recipeID, Recipe) => {
+const getByID = asyncHandler(async (recipeID, userID, Recipe) => {
     await Recipe.sync()
-    const recipe = await Recipe.findOne({ where: { id: recipeID } })
+    const recipe = await Recipe.findOne({ where: { id: recipeID, userID: userID } })
```
Comment (dntrng) [inline server/utils/recipeUtils.js:20]: recipeID is already unique so userID is not needed here

File: server/utils/recipeUtils.js [hunk 2]
```diff
@@ -15,13 +15,57 @@ const getByName = asyncHandler(async (recipeName, userID, Recipe) => {
...
+		},
+		force: true
+	}).then(() => {
+		return true
+	}, () => {
+		return false
+	})
+})
+
+const updateRecipeByID = asyncHandler(async (recipeID, recipeName, description, instructions, imageURL, userID, Recipe) => {
```
Comment (dntrng) [inline server/utils/recipeUtils.js:53]: this also doesnt need userID

File: server/utils/recipeUtils.js (hunk 1 above)
Comment (TuanDinhLe) [reply server/utils/recipeUtils.js]: Resolved.

File: server/utils/recipeUtils.js (hunk 2 above)
Comment (TuanDinhLe) [reply server/utils/recipeUtils.js]: Resolved.

**Review by dntrng — APPROVED** [review]

**Pull request merged**